source/swap/contracts/Swap.sol

```diff
@@ -54,16 +56,21 @@ contract Swap is ISwap {
   // Mapping of signer addresses to an optionally set minimum valid nonce
   mapping (address => uint256) public signerMinimumNonce;
 
+  // contains registry of kind to asset types
+  TransferHandlerRegistry public registry;
+
   /**
     * @notice Contract Constructor
     * @dev Sets domain for signature validation (EIP-712)
+    * @param _registry TokenRegistry contract
```

Comment:
After the parameter name, we then put the type before the description. So
`swapRegistry TransferHandlerRegistry Contract to ...`